install/kubernetes/helm/istio/charts/servicegraph/templates/service.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: servicegraph
+  name: {{ template "servicegraph.fullname" . }}
```

Comment:
Thanks @waret  One question is why do we need to updated this? I found that most of the `metadata.name` for Istio chart are using hardcode name, I think this does not impact any function.

Also pls sign the CLA first.

Reply:
ingress.yaml文件中引用的serviceName用的是 {{ template "servicegraph.fullname" . }}，这样如果通过ingress无法访问servicegraph服务。不过，这样来看，修改ingress.yaml里的serviceName是比较合适的，谢谢。

Reply:
@Waret welcome to the Istio community!  Would you mind translating your response to English?

I think if this change were to be made, it would need  to be made across all services, have no upgrade impact (for both helm template and helm upgrade modes of operation).

Reply:
Does the `name` impact? I think what impact is `labels`?

Reply:
Yes, it does. Please refer to [The Ingress Resource](https://kubernetes.io/docs/concepts/services-networking/ingress/#the-ingress-resource).
In servicegraph/templates/ingress.yaml, `serviceName` refers to `servicegraph.fullname` for now.

Reply:
I see, thanks @waret . +1 to update `ingress.yaml` directly to use `servicegraph` as its service name.

Reply:
So I will send another PR, thanks.